.travis.yml

```diff
@@ -17,8 +18,13 @@ env:
     - TOX_ENV=django111
 matrix:
   include:
-    - python: 2.7
+    - python: 3.6
       env: TOX_ENV=quality
+    - python: 3.7
+      dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
```

Comment:
nit: what about setting dist: xenial at the toplevel instead? You could then add **3.7** to the list of supported python interpreters instead of matrix/include. If you choose this path, you may prefer to **sudo apt-get install xvfb** and **xvfb-run --server-args=-ac tox** instead of running Xvfb the way it's currently done.

Reply:
xenial is _generally available_ but is not the Travis default so I take the conservative approach of only using xenial where it is absolutely necessary.  This also has the advantage of testing our software's interactions with both trusty and xenial.